docs/user/urls.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 10
+---
+
+# Agama URLs
+
+When you want to perform an unattended installation, Agama needs to know where
```

Comment:
It's not only the Unattended installation, with https://github.com/agama-project/agama/pull/2182 the DUD can do it too, but I believe that all similar parts, such as self-update (e.g/, through OEMDRV) should support it too. As the initial implementation was for UA, it OK to mention only that for now.

Reply:
Yes, this was mostly just a heads up. Nothing more.